Flick/Extensions/Date+Shared.swift

```diff
@@ -11,6 +11,39 @@ import Foundation
 extension Date {
     // ISO 8601 representation of date with fractional seconds
     var iso8601withFractionalSeconds: String { return Formatter.iso8601withFractionalSeconds.string(from: self) }
+    
+    /// Return the date label to be displayed given comment's created date
+    func getDateLabelText(createdAt: String) -> String {
```

Comment:
lol this is a lot of formatting

Reply:
lol shit code too but maybe I will fix this next time